config/webpack/loaders/devSiteExample.js

```diff
@@ -1,34 +1,65 @@
 const path = require('path');
 const startCase = require('lodash.startcase');
+const findCss = require('./devSiteCssFinder');
 
 /**
  * Generate the example with the supplied file.
  * Don't use an arrow function
  */
-const loader = async function loader() {
+const loader = async function loader(content) {
   const callback = this.async();
 
   const exampleSource = this.resourcePath;
   const parsedResourcePath = path.parse(exampleSource);
+  const cssFileName = findCss(content);
```

Comment:
is this the css file name like: `myStyles.module` or is this a file path?

Reply:
It returns a file path, I'll update the name for clarity.

Reply:
Updated name to findCssFileName: here: [d830ef0](https://github.com/cerner/terra-dev-site/pull/281/commits/d830ef07a820fb004bc59f65cde8f20ee7052353#).